src/components/CioQuiz/index.tsx

```diff
@@ -58,6 +60,7 @@ export default function CioQuiz(props: IQuizProps) {
   if (state.quiz.requestState === RequestStates.Success) {
     return (
       <div className='cio-quiz'>
+        <style>.cio-quiz {convertPrimaryColorsToString(primaryColorStyles)}</style>
```

Comment:
Good catch! I like that idea, I'm just not sure if it would be confusing to the end user. The alternative is just doing `<div className='cio-quiz' style={primaryColorStyles}>`. I didn't want to hardcode the values using styles but maybe it's better 🤔 I'll think about this. Let me know if you think of any pros/cons for either approach

Reply:
hmm, I guess the main pro is having a class-name selector in case you want to do additional style changes, with the con being the selector might be confusing. I think your `<... style={primaryColorStyles}` is probably the way to go here. If we really wanted a selector for individually-styled Quizzes components, it should be implemented separately.

Is there a particular reason not to hardcode the values using styles?

Reply:
Not that big of an issue but it makes it a bit harder to override the styles. But that might be okay since this is already an override. I don't think people would do this and try to add more overrides (to color specifically) using CSS. Let me loop in @deomsj  as well but I think style is the way to go here

Reply:
love the discussion and ideas here, @Mudaafi and @esezen. Definitely tradeoffs at play here. I had noted considered how multiple quizzes instances with different color values could be supported with this css variable pattern. 

My advice here would be to prioritize for extensibility and simplicity, in that order. 

Having wrestled one too many otherwise-awesome OS libraries who's authors couldn't image a consumer would want to override their inline styles, I have a strong bias against inline styles for our OS libs whenever we can avoid this. While inline styles is a clear winner in terms of simplicity it fails our extensibility criteria.

From an extensibility perspective, there's a lot about @Mudaafi's idea that I like for this,  but I don't think we will see this use case enough to justify the added complexity of that this would introduce to the src code and the resulting impact on maintainability of our library. 

My preference would be to stick with the css variable pattern that I see in the current version of this PR, and add a note to our storybook docs encouraging consumers of the lib who want to style multiple quiz instances on the same page with different colors to use their own css to do so. This primary color interface is an option to fast track development for most use cases, so it feels perfectly reasonable to me that we build it in a way that supports most use cases and encourage those brave renegades out there slinging multiple quizzes on a page with different styles to take full ownership of those styles with their own css. 

Curious how that feels to you @esezen and @Mudaafi. Feel free to drop some time on our calendars if either of you would like to discuss this further :)